### ISSUE TITLE
Fail to allocate bitmap

### ISSUE THREAD

**Issue by spenu1** [python-engineer/snake-ai-pytorch#5]
when running the agent everything works as expected, however once the games/iterations start reaching higher numbers (around 100-300 commonly around 200) the program closes itself and i get a "Fail to allocate bitmap" in the visual studio code debugger. is there a solution to fix this or continue the program in the event of this failure? thank you

**Comment by Amoshades** [comment]
Me too

**Comment by ethicalhacker7192** [comment]
Yes, the solution is to get more memory, see issue #7 to do this for the GPU, i also recommend instead of using microsoft visual studio, you should get an actual installation of python [here.](https://python.org/downloads)
